packages/isomorphic/src/utils/index.ts

```diff
@@ -70,3 +70,6 @@ export const randomBytes: typeof RandomBytesFn = (size) => {
   return toUint8Array(cryptoRandomBytes(size))
 }
 /* eslint-enable func-style */
+export * from './shared'
+// eslint-disable-next-line node/prefer-global/text-encoder, node/prefer-global/text-decoder
+export { TextDecoder, TextEncoder } from 'util'
```

Comment:
Needed because I was getting errors in several places when using them directly.

Reply:
Could you add that as a comment? (Along with the specific errors you would get if you remember?)

Reply:
Addressed in 11dd21c101084f6c00ee9d606bbdb0ad1dc048e7